knodle/transformation/torch_input.py

```diff
@@ -26,3 +26,12 @@ def input_info_labels_to_tensordataset(
     input_ids_label_dataset = TensorDataset(*model_tensors, torch.from_numpy(input_info), torch.from_numpy(labels))
 
     return input_ids_label_dataset
+
+
+def dataset_to_numpy_input(model_input_x: TensorDataset) -> np.ndarray:
+    if len(model_input_x.tensors) == 1:
+        return model_input_x.tensors[0].numpy()
+    else:
+        raise ValueError(f"Selected denoising method accepts only two-dimensional encoded input features (features x "
```

Comment:
The error message does not correspond to the "if" condition. The message reports on the dimensionality of the input, however the "if" check is about how many different matrices/tensors are available.

Reply:
done